src/styles/button.scss

```diff
@@ -0,0 +1,25 @@
+.button {
+  height: $button-height;
+  padding: 0 48px;
+  box-sizing: border-box;
```

Comment:
Better idea is setting `box-sizing` globally for consistency

Please check other places too